src/core/MOM_porous_barriers.F90

```diff
@@ -57,69 +58,74 @@ subroutine por_widths(h, tv, G, GV, US, eta, pbv, eta_bt, halo_size, eta_to_m)
   IsdB = G%IsdB; IedB = G%IedB; JsdB = G%JsdB; JedB = G%JedB
```

Comment:
As coded here, A_layer and A_layer_prev seem to have units of [Z ~> m] or [m], and not [nondim].